src/frontend/src/components/common/Dropdown/_dropdown.scss

```diff
@@ -1,14 +1,21 @@
 .dropdown {
   display: flex;
   align-items: center;
-  position: relative;
+  position: relative; 
 }
 
 .dropdown-icon {
```

Comment:
This class should be removed if no longer in use.

Reply:
the pagination component still use it